nixos/modules/services/wayland/cage.nix

```
@@ -36,8 +36,18 @@ in {
     type = types.path;
     default = "${pkgs.xterm}/bin/xterm";
     defaultText = literalExpression ''"''${pkgs.xterm}/bin/xterm"'';
+    example = literalExpression ''
+      pkgs.writeShellScript "cage-program" \'\'
+        cd /home/${user}
+        exec ${pkgs.xterm}/bin/xterm
```

I think this also needs to be escaped, otherwise it is shown as `exec /nix/store/.../bin/xterm`.

```suggestion
        exec ``${pkgs.xterm}/bin/xterm
```

---

Yup that's right, build fails unless this is also escaped.